ReceiveMessage/models/ChatbotMessage.ts

```diff
@@ -1,4 +1,15 @@
-import mongoose, { Schema } from 'mongoose';
```

Comment:
idk how github reviews work, but in the readRequest function, there's an error being thrown by the Mongoose query. This is because the `id` being queried for isn't of type `string` as specified by the interface you just wrote, it's of some wacky qs type. We know that Twilio will always give us a single string in that payload, so query using `id as string`, which is casting that id query into string, and forcing it to fit into the interface.

Reply:
done, thanks for the headsup!